pt-BR/action_controller_overview.md

```diff
@@ -1091,7 +1091,7 @@ class ApplicationController < ActionController::Base
 end
 ```
 
-Of course, this example is anything but elaborate and doesn't improve on the default exception handling at all, but once you can catch all those exceptions you're free to do whatever you want with them. For example, you could create custom exception classes that will be thrown when a user doesn't have access to a certain section of your application:
+É claro, que este exemplo não é nada elaborado e não melhora muito a forma original de lidar com os erros, mas uma vez que você capture todas essas excessões você é livre para fazer o que quiser com elas. Por exemplo, você pode criar uma excessão personalizada para quando o usuário não tem acesso a uma parte da aplicação:
```

Comment:
```suggestion
É claro, que este exemplo não é nada elaborado e não melhora muito a forma original de lidar com os erros, mas uma vez que você capture todas essas exceções você é livre para fazer o que quiser com elas. Por exemplo, você pode criar uma exceção personalizada para quando o usuário não tem acesso a uma parte da aplicação:
```

Reply:
vou fazer todas, creio que não prestei atenção o bastante nos detalhes, mas todas me parecem validas

Reply:
acho que fiz uns commits meio ruins, preciso aprender a remover eles na hora da entrega... Mas em geral todos os seus comentários foram corrigidos pelo visto e achei letras comidas no meio do caminho que eu corrigi

Reply:
Então, acho que agora não precisa se preocupar com isso. :)
Na hora de fazer o merge do PR, o time faz squash dos commits. Isso significa que eles juntam todas as alterações num commit só, aí fica tudo dentro de uma unidade.

Se você quiser fazer essa organização você mesmo, dá pra fazer isso com o `git rebase`.

- Faça o pull do branch `upstream` no seu fork:

`$ git checkout master`
`$ git pull upstream`

- Depois passe pro seu branch:

`$ git checkout Action-Controller-Overview->-14.-Rescue`

- Execute:
`$ git rebase -i --autosquash master`

Pra mais informações, vc pode olhar [aqui](https://help.github.com/pt/github/using-git/using-git-rebase-on-the-command-line)

Reply:
Você também pode marcar os próximos commits como `fixup`. Isso faz com que eles sejam mesclados de forma automática na hora de fazer rebase:

`$ git commit --fixup`

https://saulosantiago.js.org/git/git-fixup-com-autosquash/